tests/jballerina-unit-test/src/test/java/org/ballerinalang/nativeimpl/jvm/tests/StaticMethods.java

```diff
@@ -392,4 +395,36 @@ public static BFuture getFutureOnly(BFuture future) {
     public static BTypedesc getTypeDescOnly(BTypedesc typeDesc) {
         return typeDesc;
     }
+
+    public static ArrayValue getValues(MapValue<BString, Long> intMap, MapValue<BString, BString> stringMap) {
+        int length = intMap.size() + stringMap.size();
+        ListInitialValueEntry[] entries = new ListInitialValueEntry[length];
+
+        int index = 0;
+
+        for (Map.Entry<BString, Long> intEntry : intMap.entrySet()) {
+            entries[index++] = new ListInitialValueEntry.ExpressionEntry(intEntry.getValue());
+        }
+
+        for (Map.Entry<BString, BString> stringEntry : stringMap.entrySet()) {
+            entries[index++] = new ListInitialValueEntry.ExpressionEntry(stringEntry.getValue());
+        }
+
+        return new ArrayValueImpl(new BArrayType(new BUnionType(new ArrayList(2) {{
```

Comment:
Can't we avoid creating this array type each time this is called?

Reply:
The size which is passed to the `BArrayType` constructor depends on the actual map value arguments, so we can't create the array type just once. We could maybe extract out the union type, but this method is just called twice via a single test, so I'm not sure if there's much to gain by doing so. WDYT?

Reply:
Ah just now I noticed that this is a test file. No need then